test/PushNotificationTests/PushNotificationTests.testdef

```diff
@@ -0,0 +1,11 @@
+{
+    "Tests": [
+        {
+            "Description": "Enabled: This module tests the push notifications component in WinAppSDK.",
```

Comment:
Why is 'Enabled' in the description?

Does `TestAll` have a `-List` command to list the tests and their status?

If not when are you adding it? :P

See `List-ProjectTemplates` in `tools\AddProject.ps1`

Reply:
Oh, the enabled was an old comment. Will remove.

Reply:
Added -List feature :)